about.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
```

Comment:
Did you know you can add a favicon inside the head to customize the little icon on the tab of your page?